xbee_android_library/src/main/java/com/digi/xbee/api/android/connection/usb/AndroidUSBInterface.java

```diff
@@ -410,14 +414,18 @@ private UsbDevice findDevice() {
 		UsbDevice usbDevice = null;
 		HashMap<String, UsbDevice> deviceList = usbManager.getDeviceList();
 		for (UsbDevice device:deviceList.values()) {
-			if (device.getVendorId() == VID) {
-				for (int pid:FTDI_PIDS) {
+			if (device.getVendorId() == VID[0]) {
+				for (int pid:UART_PIDS) {
 					if (device.getProductId() == pid) {
 						usbDevice = device;
-						logger.info("USB XBee Android device found: " + usbDevice.getDeviceName());
+						logger.info("USB XBee Android device from FTDI vendor found: " + usbDevice.getDeviceName());
 						break;
 					}
 				}
+			} else if (device.getVendorId() == VID[1] && device.getProductId() == UART_PIDS[0]){
+			        usbDevice = device;
+			        logger.info("USB XBee Android device from EXAR vendor found: " + usbDevice.getDeviceName());
```

Comment:
FYI: Add different logs to be sure that the new EXAR vendor device was found.

Reply:
Please contact contact@beechat.network to claim bounty

Reply:
Unresponsive email

Reply:
Can you share an email here and we'll contact you today? thanks in advance

Reply:
haik.hovhanisyan@gmail.com

Reply:
Unresponsive email